src/user-management/utils.js

```diff
@@ -0,0 +1,43 @@
+function addClass(element, className) {
+  element.classList.add(className);
+}
+
+function removeClass(element, className) {
+  element.classList.remove(className);
+}
+
+async function makeApiCall(
+  url,
+  method = 'get',
+  body = null,
+  credentials = 'include',
+  headers = { 'content-type': 'application/json' },
+  options = null,
+) {
+  try {
+    const response = await fetch(url, {
+      method,
+      body,
+      headers,
+      credentials,
+      ...options,
+    });
+    return response;
+  } catch (err) {
+    console.error(MESSAGE_SOMETHING_WENT_WRONG, err);
+    return err;
+  }
+}
```

Comment:
This can be a common util rather than keeping it in a separate user management util

Reply:
According to the current folder structure of this module, if I add it in common util, it will not work, I am working on fixing the folder structure in another issue, and once the folder structure will get fixed, I will add all functions which can be reused in common utils

Reply:
- [ ] ☝️ Please add ticket for the above mentioned refactor